kotlinx-nodejs/src/worker_threads.worker_threads.module_node.kt

```diff
@@ -44,77 +44,32 @@ external open class MessagePort : EventEmitter {
     open fun unref()
     open fun start()
     open fun addListener(event: String /* "close" */, listener: () -> Unit): MessagePort /* this */
-    override fun addListener(event: String, listener: (args: Any) -> Unit): MessagePort /* this */
+    override fun addListener(event: String /* "message" */, listener: (value: Any) -> Unit): MessagePort /* this */
```

Comment:
why names were changed?

Reply:
because the logic of translation changed for following code:
```ts
 addListener(event: "message", listener: (value: any) => void): this;
addListener(event: string | symbol, listener: (...args: any[]) => void): this;
```